### DISCUSSION TITLE
%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EC%B6%9C%EC%82%AC%ED%91%9C/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#4] [Comments]
# %EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EC%B6%9C%EC%82%AC%ED%91%9C/

양성빈의 블로그

https://sungbin.kr/%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EC%B6%9C%EC%82%AC%ED%91%9C/

**Comment by devmeeple** [comment]
도입과 WDD까지 흥미로운 내용이 많네요.
(이미 나눴던 대화인데도 글에서 주는 감정이란…)

성빈클럽이 궁극적으로는 아직도 회자되는 백기선님의 ‘봄싹 스터디’처럼 됐으면 좋겠습니다.

> 같이 모르는 것에 대해 깊이 있고 심도 있는  내용을 다뤘으면 하는 바람이다.

이 부분에 가장 공감하고 기대하는 부분이에요.
함께 잘 운영해봐요!
응원합니다.

**Reply by SungbinYang** [reply]
답글 감사합니다! 화이팅입니다!
